docs/source/sci_api_tutorial/assign_mutant.rst

```diff
@@ -0,0 +1,193 @@
+==============================================
+Assign Mutant
+==============================================
+
+Briefs
+==============================================
+The target for this tutorial to mutate the target protein at defined residue(s). The main function provided are ``assign_mutant()`` and ``mutate_stru()``, in which ``assign_mutant()`` assigns mutants targeted in the study. Decode the user assigned ``{pattern}`` based on the ``{stru}`` and get a list of mutants defined by a list of mutation objects each.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/mutation_assign_mutant_io.svg
+        :width: 100%
+        :alt: assign_mutant_io 
+
+Input
+------------------------------------------------
+
+``stru``
+    the target protein structure intended for mutation, represented as a  ``Structure()`` object.
+
+    .. admonition:: How to obtain
+
+        | A Strutcure() object can be obtained by these `APIs <obtaining_stru.html>`_.
+
+``pattern``
+   specifies the residue(s) targeted for mutation, need to be clarified.
+
+``chain_sync_list``
+    a list that identifies homologous chains in the multimer protein, need to be clarified.
+
+``chain_index_mapper``
+    a dictionary mapping the residue indices for each chain in the multimer protein, need to be clarified.
+
```

Comment:
Each input needs a "How to obtain" admonition. User have to know how to compile those input here. You can link them to another section if say the pattern is very complicated

Reply:
See QZ's branch for reference

Reply:
This part needs to be more detailed. After doing this you can creat links in the argument section to here to avoid repeating.